src/test/operators/recreation_test.cpp

```diff
@@ -51,7 +51,7 @@ template <typename T>
 class RecreationTestJoin : public RecreationTest {};
 
 // here we define all Join types
-using JoinTypes = ::testing::Types<JoinNestedLoop, JoinHash, JoinSortMerge>;
+using JoinTypes = ::testing::Types<JoinNestedLoop, JoinHash /*, JoinSortMerge*/>;
```

Comment:
What happened here? Why did you remove the SortMerge and why didn’t you add yours?

Reply:
We had a temporary problem with Sort Merge Join. Seems we overlooked this incidence, thanks for catching it.